tests/functional/firefox/test_android.py

```diff
@@ -15,6 +16,30 @@ def test_family_navigation(base_url, selenium):
     assert page.family_navigation.is_menu_displayed
 
 
+@pytest.mark.flaky(reruns=1)
+def test_send_to_device_sucessful_submission(base_url, selenium):
+    page = AndroidPage(base_url, selenium).open()
+    send_to_device = page.send_to_device
+    send_to_device.type_email('noreply@mozilla.com')
+    send_to_device.click_send()
+    assert send_to_device.send_successful
+
+
+@pytest.mark.nondestructive
+def test_send_to_device_fails_when_missing_required_fields(base_url, selenium):
+    page = AndroidPage(base_url, selenium).open()
+    with pytest.raises(TimeoutException):
+        page.send_to_device.click_send()
+
+
+@pytest.mark.smoke
+@pytest.mark.nondestructive
+def test_send_to_device_not_supported_locale(base_url, selenium):
+    page = AndroidPage(base_url, selenium, 'it').open()
```

Comment:
Nit: If you include the keyword for the argument it makes it more readable `locale='it'`